src/Core/Domain/Employee/ValueObject/EmployeeId.php

```diff
@@ -31,7 +31,7 @@
 /**
  * Defines Employee ID with it's constraints.
  */
-class EmployeeId
+class EmployeeId implements EmployeeIdInterface
```

Comment:
BC Break to document

Reply:
I added the details in the PR description